src/rauc-installer.c

```diff
@@ -163,35 +163,38 @@ static gpointer install_loop_thread(gpointer data)
         GBusType bus_type = (!g_strcmp0(g_getenv("DBUS_STARTER_BUS_TYPE"), "session"))
                             ? G_BUS_TYPE_SESSION : G_BUS_TYPE_SYSTEM;
         RInstaller *r_installer_proxy = NULL;
-        GError *error = NULL;
-        struct install_context *context = data;
+        g_autoptr(GError) error = NULL;
+        struct install_context *context = NULL;
+
+        g_return_val_if_fail(data, NULL);
```

Comment:
nit: in the other commits you simply did `g_return_val_if_fail(context)` which is inconsistent with this one.